src/sections/Company/Contact/contactpage.style.js

```diff
@@ -28,21 +28,30 @@ const ContactPageWrapper = styled.section`
 	}
 
 	.contact-form {
-		display: none;
-
+		display: flex;
```

Comment:
Why this is needed?

Reply:
And other changes to height m

Reply:
It's fine, the only place where it can be improved is when we are closing the form at that time the form is instantly set to `0` height and doesn't go with the transition. I think that can be achieved using animation. Can you look into that part?

Reply:
Yes .. I was also thinking to improve it.
Yes ..I'll do that. 👍🏻